front/src/main.js

```diff
@@ -35,6 +36,11 @@ new Vue({
       }
       return Promise.reject(error);
     });
+
+    if (this.$store.getters.isLoggedIn) {
+      // get userSettings data from server
```

Comment:
these comments usually means that your code is understandable

Reply:
1) it is really strange name fetchServer if you fetch actually user settings
2) i don't agree that only one method is enough and has sense to place into separate service